datasets/kz/afmrk_sanctions/crawler.py

```diff
@@ -32,8 +32,8 @@ def make_entity(context: Context, el, schema, entity_id, topics, program):
     sanction = h.make_sanction(context, entity)
     sanction.add("summary", el.findtext("./correction"))
     sanction.add("program", program)
-    listingDate = h.parse_date(el.findtext("./added_to_list"), FORMATS)
-    note_date = added_date_from_note(el.findtext("./correction"))
+    listingDate = h.extract_date(context.dataset, el.findtext("./added_to_list"))
+    note_date = added_date_from_note(context, el.findtext("./correction"))
     sanction.add("listingDate", listingDate or note_date)
```

Comment:
are we relying on the parsing to make listingDate falsy or is the unparsed value already falsy?

line 37 can already be apply, and depending on the above, we might also be able to skip the extract_date on the new line 35.